src/ui/RedditListCell.tsx

```diff
@@ -1,14 +1,11 @@
-import {Composite, ImageView, TextView} from 'tabris';
-import {component, property} from 'tabris-decorators';
-import {RedditPostData} from '../common';
+import {Composite, ImageView, TextView, Properties} from 'tabris';
+import {component, Cell} from 'tabris-decorators';
 
-@component export default class RedditListCell extends Composite {
+@component export default class RedditListCell extends Cell {
```

Comment:
The `Cell` here and in the RedditGalleryCell should include the generic type, i.e. Cell<RedditPost>.
You should also set the `itemType` property, that enables an additional type check.
See if it works like this

`itemType = RedditPost;`

placed abovel the constructor